updated Fedora version

boredsquirrel opened the pull request.

boredsquirrel committed:
updated Fedora version

boredsquirrel committed:
Update to F38, fixed DNS overwrite, removed Google, added Fallback

boredsquirrel committed:
Update README.md

boredsquirrel commented:
[Here is a list of trusted DNS providers](https://www.privacy-handbuch.de/handbuch_93d.htm)

I would never include a "use Google, it works" into an automated script.

Also you should never change the resolv.conf and also add overwrites only through an overwrite file, which I did.

elvisisvan commented:
this is good, why hasn't the repo owner accepted it yet?!

elvisisvan commented:
i checked out your code using `gh pr checkout 65` command and `cp termux-fedora.sh ~` then `sh ~/termux-fedora.sh` , got these errors: 

mkdir: invalid option -- 'P'
![Screenshot_20231217-163240](https://github.com/nmilosev/termux-fedora/assets/69122770/2a230f0f-228b-4a55-b59a-d4eaae015073)
i tried again with `mkdir -p` and it works, it's the capitalization of letter "P" that caused the error

`dnf update` returns "dnf: command not found" 
![Screenshot_20231217-163354](https://github.com/nmilosev/termux-fedora/assets/69122770/4d93e8f0-6b6b-48d8-9ff5-ce51143ecfa7)

boredsquirrel committed:
updates

updated the image to f39

fixed the mkdir -p spello

todo: add a rawhide source (Docker hub needs auth token)

boredsquirrel commented:
Fixed the spelling issue and upgraded the image to F39.

It would be great to support docker hub, as they have a rawhide image too (saving the upgrades if you are lazy).

docker hub needs an auth token:

```
# get token
export TOKEN=\
"$(curl \
--silent \
--header 'GET' \
"https://auth.docker.io/token?service=registry.docker.io&scope=repository:library/ubuntu:pull" \
| jq -r '.token' \
)"
```

https://devops.stackexchange.com/questions/2731/downloading-docker-images-from-docker-hub-without-using-docker

this describes the complete (not automated) steps to download a docker image without docker. Should work fine, but using something like Ublue does should make the system way better.

elvisisvan commented:
thanks for your suggestions, i actually was planning to use qbittorrent not docker, can you install and run qbittorrent? 

on my desktop i run fedora 38 kde spin and every time i run `lsof -i -P -n | grep LISTEN` terminal will display a list of programs that have daemon running and are listening to which port, qbittorrent will listen on port 8117 if it is running, i want to replicate that behavior, can we do that?

juhp commented:
I don't see the need to use docker for rawhide.

Would be nice to update the versions in the readme too to make it all consistent
